.pre-commit-hooks.yaml

```
@@ -108,3 +108,5 @@
   description: Runs terrascan on Terraform templates.
   language: script
   entry: terrascan.sh
+  files: \.tf$
+  exclude: \.terraform\/.*$
```

Improve hook speed from 2m44s to 1m 03s

```suggestion
  exclude: \.terraform\/.*$
  require_serial: true
```